src/governance-vote.cpp

```diff
@@ -11,59 +11,52 @@
 
 std::string CGovernanceVoting::ConvertOutcomeToString(vote_outcome_enum_t nOutcome)
 {
-    switch (nOutcome) {
-    case VOTE_OUTCOME_NONE:
-        return "NONE";
-        break;
-    case VOTE_OUTCOME_YES:
-        return "YES";
-        break;
-    case VOTE_OUTCOME_NO:
-        return "NO";
-        break;
-    case VOTE_OUTCOME_ABSTAIN:
-        return "ABSTAIN";
-        break;
+    static const std::map<vote_signal_enum_t, std::string> mapOutcomeString = {
+        { VOTE_OUTCOME_NONE, "none" },
+        { VOTE_OUTCOME_YES, "yes" },
+        { VOTE_OUTCOME_NO, "no" },
+        { VOTE_OUTCOME_ABSTAIN, "abstain" } };
+
+    const auto& it = mapOutcomeString.find(nOutcome);
+    if (it == mapOutcomeString.end()) {
+        LogPrintf("CGovernanceVoting::%s -- ERROR: Unknown outcome %s\n", __func__, nOutcome);
+        return "error";
     }
-    return "error";
+    return it->second;
 }
 
 std::string CGovernanceVoting::ConvertSignalToString(vote_signal_enum_t nSignal)
 {
-    std::string strReturn = "NONE";
-    switch (nSignal) {
-    case VOTE_SIGNAL_NONE:
-        strReturn = "NONE";
-        break;
-    case VOTE_SIGNAL_FUNDING:
-        strReturn = "FUNDING";
-        break;
-    case VOTE_SIGNAL_VALID:
-        strReturn = "VALID";
-        break;
-    case VOTE_SIGNAL_DELETE:
-        strReturn = "DELETE";
-        break;
-    case VOTE_SIGNAL_ENDORSED:
-        strReturn = "ENDORSED";
-        break;
+    static const std::map<std::string, vote_signal_enum_t> mapSignalsString = {
```

Comment:
`std::map<vote_signal_enum_t, std::string>`
and name `mapSignalString` instead of `mapSignalsString`?